src/pages/home/report/ReportActionItem.js

```diff
@@ -363,6 +363,7 @@ class ReportActionItem extends Component {
             );
         }
 
+        const isReportValid = !lodashGet(this.props.report, 'errorFields.addWorkspaceRoom') && !lodashGet(this.props.report, 'errorFields.createChat');
```

Comment:
I don't love this variable name. Also, why do we need to check for `errorFields.addWorkspaceRoom`? I thought the problem of the redundant message only occurs when there is another chat that already exists (i.e. `errorFields.createChat` has the error)

Reply:
Ah you're right, I found that out a little late [here](https://github.com/Expensify/App/pull/19188#issuecomment-1553144256). I'll remove the variable completely.

Reply:
Actually the `errorFields.addWorkspaceRoom` might be required. I'll have to check and let you know. But the reason I initially added this is because the `ReportActionItemCreated` component displays it.

Reply:
Is this error something that's happening for the `addWorkspaceRoom`? If we're not seeing it in a case, let's not code for it. Let's just fix what is broken and move on, IMO.

Reply:
I've confirmed that the `addWorkspaceRoom` error field is not required here just as you said - [Slack Conversation](https://app.slack.com/client/T03SC9DTT/C01GTK53T8Q/thread/C01GTK53T8Q-1684812618.894539).